lib/babel.js

```diff
@@ -70,11 +68,43 @@ if (inputSourceMap) {
 }
 
 // include test file
-var transpiled = babel.transformFileSync(testPath, options);
-sourceMapCache[testPath] = transpiled.map;
-requireFromString(transpiled.code, testPath, {
-	appendPaths: module.paths
-});
+var cachePath = hasha(testPath);
+var hashPath = hasha(testPath) + '_hash';
+
+var prevHash = cache.getSync(hashPath, {encoding: 'utf8'});
+var currHash = hasha.fromFileSync(testPath);
+
+if (prevHash === currHash) {
+	var cached = JSON.parse(cache.getSync(cachePath));
```

Comment:
actually scratch that. I think `power-assert` may include the filename. In the transform.

@twada?